modules/gitops-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-3_{context}"]
+= Release notes for {gitops-title} 1.8.3
+
+{gitops-title} 1.8.3 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="errata-updates-1-8-3_{context}"]
+== Errata updates
+
+=== RHEA-2023:112944 and RHBA-2023:112945 - {gitops-title} 1.8.3 security update advisory 
+
+Issued: 2023-04-17
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
+* link:https://access.redhat.com/errata/RHBA-2023:112945[RHBA-2023:112945]
```

Comment:
@reginapizza  and @iam-veeramalla: The Errata links are not accessible from the preview page. I think they are still not fixed? don't know if this is some permission issue?
https://access.redhat.com/errata/RHEA-2023:112944

No idea why. I need the errata links for them.

Any help here is much appreciated. TIA : )

Reply:
@Srivaralakshmi LGTM from my side once we get appropriate links

Reply:
FYI, I am providing the following links:

- https://access.redhat.com/errata/RHBA-2023:112946
- https://access.redhat.com/errata/RHBA-2023:112945

These links go to the Customer Portal, not to the Errata Tool. The Errata Tool is internal only.
Users wouldn't even be able to access it.
Now you are seeing **404 Page not found** as they are not released yet. See their status. It says NEW FILES. They will only become visible on the CP when released, i.e. when their status is SHIPPED LIVE.
When released, the errata will have URLs like this: https://access.redhat.com/errata/RHBA-2023:112945.